recipe/meta.yaml

```diff
@@ -28,13 +28,13 @@ requirements:
     - pip
     - cython
     - numpy
-    - libgdal
+    - gdal
     - proj
     - setuptools
   run:
```

Comment:
Can you add an explicit pin on `numpy>=1.24`, xref https://github.com/rasterio/rasterio/pull/3166.